app/js/components/App.jsx

```diff
@@ -9,13 +9,26 @@
  * under the License.
  * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
  */
+
 import React, {Component} from 'react';
 import {Header} from './common/Header';
 import PageComponent from './page/pageComponent';
 import BreadCrumbComponent from './breadCrumb/breadCrumbComponent';
 import CohortTable from '../components/common/table';
 
+/**
+ * The App Component class
+ * 
+ * @class App
+ * @extends {Component}
+ */
 class App extends Component {
+
+  /**
+   * Creates an instance of App.
```

Comment:
Same problem here as above. Describing a language term is not documentation because any one who does ReactJS should know what this does. In other words, comments are not about teaching some one a language. Does it make sense? :)